src/v/redpanda/application.cc

```diff
@@ -83,6 +83,7 @@
 #include "config/seed_server.h"
 #include "config/types.h"
 #include "crypto/ossl_context_service.h"
+#include "debug_bundle/debug_bundle_service.h"
```

Comment:
nitpick: could we get away with forward declarations here?

Reply:
> nitpick: could we get away with forward declarations here?

I don't think so, calls to `start` and `stop` are required.